src/data/redux.test.jsx

```diff
@@ -250,25 +348,48 @@ describe('Data layer integration tests', () => {
     const readyToSubmitAttempt = Factory.build('attempt', { attempt_status: ExamStatus.READY_TO_SUBMIT });
     const readyToSubmitExam = Factory.build('exam', { attempt: readyToSubmitAttempt });
 
-    it('Should stop exam, and update attempt and exam', async () => {
-      axiosMock.onGet(fetchExamAttemptsDataUrl).replyOnce(200, { exam: readyToSubmitExam, active_attempt: {} });
-      axiosMock.onGet(fetchExamAttemptsDataUrl).reply(200, { exam, active_attempt: attempt });
-      axiosMock.onPost(updateAttemptStatusUrl).reply(200, { exam_attempt_id: attempt.attempt_id });
```

Comment:
in a few cases where the mocked method isn't even correct but because we're never asserting it's called the test was passing.